externals/ssl/src/CVMFS-CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+include ("${EXTERNALS_LIB_LOCATION}/CMake-Register_External_Lib.txt")
+
+set (SSL_ARCHIVE "${SSL_BUILTIN_LOCATION}/ssl_install/lib/libssl.a;${SSL_BUILTIN_LOCATION}/ssl_install/lib/libcrypto.a")
```

Comment:
I think in cvmfs/CMakeLists.txt we'd need to add libssl as a dependency on OS X everywhere where we need `OPENSSL_LIBRARIES`.  We'd also need to add `SSL_ARCHIVE` where we need `OPENSSL_LIBRARIES`.